### PR TITLE
Added null checks to AppUploader constructor

### LINKED ISSUE #6: Add Better Error Handling and Common Checks in AppUploader Constructor
The [`AppUploader`](https://github.com/LambdaTest/lambdatest-gradle-plugin/blob/main/src/main/java/io/github/lambdatest/gradle/AppUploader.java) constructor lacks proper error handling and common checks (e.g., null checks), which may lead to runtime issues.

Adding checks and proper error handling will improve the robustness of this class.

### PR DESCRIPTION
**Pull request by Taz03**
closes #6

### CODE REVIEW

File: src/main/java/io/github/lambdatest/gradle/AppUploader.java [hunk 1]
```diff
@@ -13,6 +13,9 @@ public class AppUploader {
     private String appFilePath;
 
     public AppUploader(String username, String accessKey, String appFilePath) {
+        if (username == null || accessKey == null || appFilePath == null)
```
Comment (harsha509) [inline src/main/java/io/github/lambdatest/gradle/AppUploader.java:16]: HI @Taz03 ,

Thank you for the pr.
Having a generic message is good, but thinking if we can improve more.  below is my sample suggestion, let me know your thoughts!
```suggestion
 if (username == null) throw new IllegalArgumentException("Username cannot be null");
 if (accessKey == null) throw new IllegalArgumentException("Access Key cannot be null");
 if (appFilePath == null) throw new IllegalArgumentException("App File Path cannot be null");
```

Thanks,
Sri!

File: src/main/java/io/github/lambdatest/gradle/AppUploader.java [hunk 2]
```diff
@@ -13,6 +13,9 @@ public class AppUploader {
...
+            throw new IllegalArgumentException("Arguments cannot be null");
+
```
[the remark this answers is not in the document]
Comment (Delta456) [reply src/main/java/io/github/lambdatest/gradle/AppUploader.java:18]: I think @harsha509 review above is better and also covers that.

File: src/main/java/io/github/lambdatest/gradle/AppUploader.java (hunk 1 above)
Comment (Delta456) [reply src/main/java/io/github/lambdatest/gradle/AppUploader.java:16]: I feel we should go for this as well. It is more accurate.

**Review by navin772 — APPROVED** [review]

**Review by harsha509 — APPROVED** [review]
Thank you @Taz03 !

LGTM!

**Review by Delta456 — APPROVED** [review]
Congratulations and amazing work @Taz03! 

You have done very well 🚀

**Pull request merged**